[Update] Self Hosted Install

Rajakavitha1 opened the pull request:
# Description

- Used Tabs for different Linux systems on one page.
- Fixed the install version for timescale db  with apt 
- The PGDG ubuntu versioning not updated because the warning is now specific and states `apt` package manager maintained by Debian or Ubuntu
Fixes: https://github.com/timescale/docs/issues/1905,  https://github.com/timescale/docs/issues/1975

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

Loquacity commented:
Because you've got all three locations pointing to the same page, you end up with this weirdness when you're on the page: 
![image](https://user-images.githubusercontent.com/3914967/213960313-ed63979b-ed83-4d01-96a9-5a8bb278f09f.png)

It might be better to reconfigure the index page to just have one link to Linux, rather than directing all three Linux flavours to one place?

Loquacity commented:
Tabs appear to be working well, but I think we can probably drop the 'based' from the tab labels. What do you think? 
![image](https://user-images.githubusercontent.com/3914967/213960450-9e9b8bc2-9c86-49ca-936d-8ffd9c6df664.png)


@charislam not sure if it's my browser, but the shading on the selected tab is very, very pale. Can we easily tweak that a shade or two?

Loquacity commented:
On this one, the heading size is different for the first two tabs than it is for the Arch tab:

![image](https://user-images.githubusercontent.com/3914967/213960571-6ea00b5e-411a-4113-8d62-ed02658ef291.png)

![image](https://user-images.githubusercontent.com/3914967/213960586-5ee9cbf3-b0d6-4e1b-bb70-3c9f46167303.png)

Rajakavitha1 commented:
> Because you've got all three locations pointing to the same page, you end up with this weirdness when you're on the page: ![image](https://user-images.githubusercontent.com/3914967/213960313-ed63979b-ed83-4d01-96a9-5a8bb278f09f.png)
> 
> It might be better to reconfigure the index page to just have one link to Linux, rather than directing all three Linux flavours to one place?

Updated the index page!!! added the Linux icon too...

Rajakavitha1 commented:
> Tabs appear to be working well, but I think we can probably drop the 'based' from the tab labels. What do you think? ![image](https://user-images.githubusercontent.com/3914967/213960450-9e9b8bc2-9c86-49ca-936d-8ffd9c6df664.png)
> 
> @charislam not sure if it's my browser, but the shading on the selected tab is very, very pale. Can we easily tweak that a shade or two?

removed "based"

charislam commented:
> Tabs appear to be working well, but I think we can probably drop the 'based' from the tab labels. What do you think? ![image](https://user-images.githubusercontent.com/3914967/213960450-9e9b8bc2-9c86-49ca-936d-8ffd9c6df664.png)
> 
> @charislam not sure if it's my browser, but the shading on the selected tab is very, very pale. Can we easily tweak that a shade or two?

We can if we really want. I'm a little hesitant to do it because (1) I directly copied the styles from the product team's design library, so this version is "design-approved" and (2) it will decrease contrast between the text and the tab background, which I think is more important, since the selected tab is already highlighted by text/border color and text/border weight.

That's the context, but if you still want me to do it, I can darken the background!

charislam commented on _partials/_psql-installation-linux.md:
```diff
@@ -0,0 +1,74 @@
+### Install psql on Linux
```
How many times is this partial used, Rajie? I ony see it once, but perhaps there are future use cases on other pages that I'm unaware of.

If we don't use a partial at least twice, I would prefer to inline it, because each partial has a slight negative impact on our site performance

Rajakavitha1 replied:
@charislam I plan to use the partial here https://docs.timescale.com/timescaledb/latest/how-to-guides/connecting/psql/. Did not want to add too many files to this PR. I will create a new PR later.

charislam commented on install/installation-linux.md:
```diff
@@ -0,0 +1,525 @@
+
+import Linux from "versionContent/_partials/_psql-installation-linux.mdx";
+
+## Install TimescaleDB on Linux
```
```suggestion
# Install TimescaleDB on Linux
```

Loquacity commented:
> > Tabs appear to be working well, but I think we can probably drop the 'based' from the tab labels. What do you think? ![image](https://user-images.githubusercontent.com/3914967/213960450-9e9b8bc2-9c86-49ca-936d-8ffd9c6df664.png)
> > @charislam not sure if it's my browser, but the shading on the selected tab is very, very pale. Can we easily tweak that a shade or two?
> 
> We can if we really want. I'm a little hesitant to do it because (1) I directly copied the styles from the product team's design library, so this version is "design-approved" and (2) it will decrease contrast between the text and the tab background, which I think is more important, since the selected tab is already highlighted by text/border color and text/border weight.
> 
> That's the context, but if you still want me to do it, I can darken the background!

No, I think the reasoning is sound. Leave it as-is, I'll stop complaining 😂

Loquacity commented:
This is really coming together! Nice work 😁

Loquacity commented:
We probably need to do some redirects for this, too, since we're eliminating the distro-specific pages ...

charislam commented on install/installation-linux.md:
```diff
@@ -0,0 +1,525 @@
+
+</Tab>
+
+<Tab title="Archlinux">
```
```suggestion
<Tab title="ArchLinux">
```

charislam commented on install/installation-linux.md:
```diff
@@ -0,0 +1,525 @@
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on archlinux-based systems
```
```suggestion
### Installing self-hosted TimescaleDB on ArchLinux-based systems
```

charislam commented on _partials/_psql-installation-linux.md:
```diff
@@ -0,0 +1,74 @@
+
+<procedure>
+
+#### Installing psql using the apt package manager
```
```suggestion
### Installing psql using the apt package manager
```

charislam commented on _partials/_psql-installation-linux.md:
```diff
@@ -0,0 +1,74 @@
+
+<procedure>
+
+#### Installing psql using the yum package manager
```
```suggestion
### Installing psql using the yum package manager
```

jonatas reviewed (approved):
Looks good to me! I haven't tested manually as I'm on Mac.

Loquacity commented on install/installation-linux.md:
````diff
@@ -0,0 +1,525 @@
+    yum install timescaledb-2-postgresql-14
+    ```
+
+    <!-- hack until we have bandwidth to rewrite this linting rule -->
````
Add it to the [standing issue](https://github.com/timescale/docs/issues/1369) ... ?
